inc/lift/Request.h

```diff
@@ -74,8 +79,27 @@ class Request {
      * @param timeout The timeout for the request.
      * @return True if the timeout was set.
      */
-    auto SetTimeout(
+    auto SetConnectionTimeout(
```

Comment:
What about other kinds of timeout (send/receive)?

Reply:
Changed to SetCurlTimeout, though not completely sold on the name.
The idea is that users can set either just the curl timeout or both the curl timeout and response wait time, with the intention being that the response wait time is significantly shorter than the curl timeout in order to preserve keep alive connections.